src/ankita/assignment/array/MissingNumber.java

```diff
@@ -0,0 +1,35 @@
+/*Program 1: Find the missing number in integer array of 1 to 10?
+Hint: You have given an integer array which contains numbers from 1 to 10 
+but exactly one number is missing, you need to write a Java program to find that missing number in an array.
+*/
+package ankita.assignment.array;
+
+import java.util.Arrays;
+
+public class MissingNumber {
+	void findMissingNumber(int[] arr) {
+
+		int[] arr1 = { 1, 2, 3, 4, 5, 6, 7, 8, 9, 10 };
```

Comment:
this is not correct way
Please refer this code and raise PR for same
https://github.com/KrishnaKTechnocredits/JAVATechnoOct2020/pull/1031/files